### DISCUSSION TITLE
Old Follow Point Behaviour

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#23825] [Ideas] [answered]
I use a modified version of the 2013/2014 skin and have gotten used to the old follow point behaviour. I looked for a way to make it such that follow points don't push out from the last note and instead just fade in (like they do in stable), but I was unable to find an option that does this.

**Comment by peppy** [comment, marked as answer]
Tracking at https://github.com/ppy/osu/issues/23831.

**Comment by Axilotl17** [comment]


**Reply by Axilotl17** [reply]
oops wrong discussion
